application-tour-ui/src/main/resources/TourCode/StepsListing.xml

```diff
@@ -101,7 +100,9 @@
                      $!placement
                   &lt;/td&gt;
                   &lt;td&gt;
-                     $!targetPage
+                     $!targetPage &lt;br&gt;
+                     $!action &lt;br&gt;
+                     $!queryString
```

Comment:
In edit mode? The edit page loops over the class properties so I didn't have to change anything.